geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/pubsub/SubscriptionsIntegrationTest.java

```diff
@@ -53,49 +54,53 @@
     };
```

Comment:
The `functionSpinner()` callable above this line is never used and can be removed. The same is true of the `ExecutorServiceRule`.

Reply:
yes